docs/Terminology.rst

```diff
@@ -0,0 +1,25 @@
+.. _Observation_Terminology:
+
+***************
+Terminology
+***************
+
+Scheduling
+==================
+Determined during the python in astronomy conference:
+
+* **observing block** (OB): the minimum schedulable block
+* **priority**: number assigned to the **observing block** by the user (or TAC) that 
```

Comment:
I think it is a good use of the term--that's how we define it at Subaru.  The user can provide priorities to help inform the scheduler of the desirability of one OB vs another if both are being considered for the same slot.  See my comment below for how we define TAC score.